portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,4 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+//console.log("Why tf is this not working?")
```

Comment:
:) I'm guessing you figured it out. Can remove debug statements since they've probably already served their purpose.